kolibri/plugins/html5_viewer/assets/src/views/Html5AppRendererIndex.vue

```diff
@@ -73,6 +73,15 @@
       this.$emit('startTracking');
       this.startTime = now();
       this.pollProgress();
+      let self = this;
+      this.hashi.on(events.CONTENTLOADED, function(data) {
+        // The HTML5 renderer has min and max heights set, so exceedingly low or height height values
+        // will not lead to problematic rendering scenarios. Consequently, this also means that it is only
+        // giving us the ability to stretch at most a couple hundred pixels height-wise.
+        if (data.offsetHeight && data.offsetHeight > 0) {
+          self.$refs.html5Renderer.$refs.fullscreen.style.height = data.offsetHeight + 'px';
```

Comment:
rather than using nested `$refs` and modifying `.style.height`, it would be a bit more conventional to add a prop to `CoreFullscreen` and pass in a `height` if possible.

For example:

```html
<template>
  <div
    ref="fullscreen"
    :class="fullscreenClass"
    :style="{ height: `${height} px` }"
    allowfullscreen
  >
    <slot></slot>
  </div>
</template>

<script>
  // ...
    props: {
      height: { type: Number, required: false },
    }
   // ...
```